coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -127,6 +148,37 @@ def metadata(self):
         """
         return self._metadata
 
+    @property
+    def class_padding(self):
+        """
+        A namedtuple consisting of values about blank lines before and after
+        the documentation of ``docstring_type`` class.
+
+        These values are official standard of following blank lines before and
+        the documentation of ``docstring_type`` class.
+        """
+        return self._class_padding
+
+    @property
+    def function_padding(self):
+        """
+        A namedtuple consisting of values about blank lines before and after
+        the documentation of ``docstring_type`` function.
+
+        These values are official standard of following blank lines before and
+        the documentation of ``docstring_type`` function.
+        """
+        return self._function_padding
+
+    @property
+    def docstring_type_regex(self):
```

Comment:
As everything we are fetching from `.coalang` is a standard. And we don't want anyone to set those up manually. Hence setting them as getters :)
PS- Even the previous things like markers, comment's signature....etc we are fetching from `.coalang` are introduced in DocstyleDefinition as getters.